osu.Game/Skinning/LegacySkin.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.IO;
+using System.Linq;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Sample;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Framework.IO.Stores;
+
+namespace osu.Game.Skinning
+{
+    public class LegacySkin : Skin
+    {
+        private readonly TextureStore textures;
+
+        private readonly SampleManager samples;
+
+        public LegacySkin(SkinInfo skin, IResourceStore<byte[]> storage, AudioManager audioManager)
+            : base(skin)
+        {
+            var audioStore = new ResourceStore<byte[]>(storage);
+
+            samples = audioManager.GetSampleManager(audioStore);

```

Comment:
Isn't this the same as `GetSampleManager(storage)`? Is there a reason we have a separate resourcestore here?

Reply:
no, got left over from some previous code